src/database/common.c

```diff
@@ -32,13 +32,12 @@ long int lastdbindex = 0;
 
 void _dbclose(sqlite3 **db, const char *func, const int line, const char *file)
 {
-	if(config.debug & DEBUG_DATABASE)
-		logg("Closing FTL database in %s() (%s:%i)", func, file, line);
+		log_debug(DEBUG_DATABASE, "Closing FTL database in %s() (%s:%i)", func, file, line);
```

Comment:
Please fix indentation here, too.
```suggestion
	log_debug(DEBUG_DATABASE, "Closing FTL database in %s() (%s:%i)", func, file, line);
```